code/go/0chain.net/smartcontract/benchmark/benchmark.go

```diff
@@ -1,6 +1,7 @@
 package benchmark
 
 import (
+	"0chain.net/core/common"
```

Comment:
goimports to format

Reply:
strange, why it is not run automatically in goland

Reply:
ah it used gofmt and it gives different result, changed to goimports